### ISSUE TITLE
Client crashes when looking up any item uses in NEI unless I go into single player first

### ISSUE THREAD

**Issue by Mikki-m8** [ReikaKalseki/Reika_Mods_Issues#1710]
I have about 100 of these crash logs that all look essentially the same. 

Either like this: https://pastebin.com/YbgHqdFF
(Optifine + BetterFPS + Fastcraft are removed in this one I think, typically I have them on but get the same crash)

or like this: https://pastebin.com/wj5DXHkv

I know this one doesn't say anything about your mods so I thought it was an issue with NEI Integration. Whenever I try to dump fluids the dump file is empty. I changed `B:"Fluid Registry"` to false/true (can't remember what it was originally tbh). Still crashed. With NEI Integrations disabled and also with all NEI addons disabled, I get this crash log instead: https://pastebin.com/Wk2mkWDK

I thought it might have been because I was right clicking the items to see the uses, as if I was going to give myself one of them, but it crashes if I press U or Shift+R as well. 

I've seen a similar issue here but that was related to Forestry, I don't have Forestry installed and afaik no other mod I have adds ethanol. Maybe AppleMilkTea has a Fermenter, but I'm not sure. I thought it might have been UniDict since it has Forestry support, but I have the same issue with Forestry support disabled and UniDict completely disabled. 

In DragonAPI.cfg, I changed `B:"Disallow Null-Item ItemStacks to Prevent Crashes"` to false, same crash. 

I've done a bunch of other stuff but I've forgotten what. The only thing that stops the crashes is removing RotaryCraft + addons (kept DragonAPI), or loading into single player first. And I found the latter completely by accident. My friend doesn't have a good enough computer to do that. 

I'll try setting up a server later and going through every mod to find conflicts, but that's a hassle. I was hoping I could get some help here before I resort to that.

**Comment by ReikaKalseki** [comment]
Something is wrong with your fluid registry. Try running /dumpids fluid client both before and after entering a singleplayer world. And on the server, try /dumpids fluid server.

**Comment by Mikki-m8** [comment]
I went on the server first and ran the client command, got a fatal error and kicked back to menu: https://pastebin.com/ENAB0vYS
Went into single player and the same thing happened: https://pastebin.com/YtmvNpAi

Restarted and went into single player first this time, it dumped them without error. Just in case it helps, here they are: https://pastebin.com/5NCMVHzm
then I went back to multiplayer, the command didn't crash me but it didn't dump any fluids either. This was in the log: https://pastebin.com/2EEaKs1K

Running /dumpids fluid server on the server gives me the same as the client single player one: https://pastebin.com/br2gkwZL

I have no idea what's going on

**Comment by Mikki-m8** [comment]
Wait, I forgot I enabled NEI Integration and other addons again. With them all disabled I don't get the fatal error when I dump fluids on the client, but I still crash if I look up the uses for any item. And the fluid dumps match too.

Went into single player first, dumped fluids fine. Went into multiplayer, dumped fluids fine. Tried to look up the uses for an item and crashed. https://pastebin.com/8Y7QCbwf

Now in this log it says something about creating mulch from Highlands blocks which is when I realised I don't have Highlands on the server. Removed Highlands from my client and it _seems_ to have stopped the crashing.

So yeah, user error. Whoops.
